transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -548,8 +661,8 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng, S: RingSigner + ?Sized>(
     // range proofs is present when they are.
     let (range_proof, range_proofs) = if !block_version.mixed_transactions_are_supported() {
         // The implicit fee output is omitted from the range proof because it is known.
-        let generator = generator_cache.get(fee.token_id);
 
+        let generator = generator_cache.get(fee.token_id);
```

Comment:
i think this is just noise in the diff, can we move this line back to the comment that is attached to it?